app/src/main/java/com/example/myweatherapp/mvp/presenter/MyWeatherAppPresenter.kt

```diff
@@ -9,8 +11,13 @@ class MyWeatherAppPresenter(
     MyWeatherAppContract.MyWeatherAppPresenter {
 
     override fun getWeatherForecast() {
-        view.showData()
-        model.getData()
+        model.getData(CITY)
+            .subscribeOn(Schedulers.io())
+            .observeOn(AndroidSchedulers.mainThread())
+            .subscribe { data -> view.showData(data) }
```

Comment:
add an error callback because, if you are without a network connection, this will crash

Reply:
Done thx